src/Microsoft.OpenApi/Microsoft.OpenApi.csproj

```diff
@@ -1,6 +1,6 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
     <PropertyGroup>
-        <TargetFrameworks>netstandard2.0</TargetFrameworks>
+        <TargetFrameworks>net6.0</TargetFrameworks>
```

Comment:
this would mean we drop support for anything else. No more dotnet framework , just net 6 and up.
I personally don't have issues with that, I just wanted to call it out.
If we go this route, we should align the openapi.odata lib as well https://github.com/microsoft/OpenAPI.NET.OData/issues/139

The alternative would be (for this lib and the readers lib) to cross compile for `netstandard2.0;net462` to keep compatibility with .net framework folks if there's no new API we're using.

Reply:
Thanks for pointing this out. I think the safest outcome would be to support cross compilation. Resolving..